conda/resolve.py

```diff
@@ -35,31 +35,39 @@
 
 _sat_solvers = odict([
     (SatSolverChoice.PYCOSAT, PycoSatSolver),
-    (SatSolverChoice.PYCRYPTOSAT, CryptoMiniSatSolver),
+    (SatSolverChoice.PYCRYPTOSAT, PyCryptoSatSolver),
     (SatSolverChoice.PYSAT, PySatSolver),
 ])
 
 
 @memoize
 def _get_sat_solver_cls(sat_solver_choice=SatSolverChoice.PYCOSAT):
-    cls = _sat_solvers[sat_solver_choice]
+    def try_out_solver(sat_solver):
+        c = Clauses(sat_solver=sat_solver)
+        required = {c.new_var(), c.new_var()}
+        c.Require(c.And, *required)
+        solution = set(c.sat())
```

Comment:
We use `Clauses.sat` here instead of `SatSolver.run` because we don't expose the `SatSolver` classes to `resolve` anymore.